common/src/main/java/me/shedaniel/architectury/event/events/client/ClientScreenInputEvent.java

```diff
@@ -29,46 +29,166 @@
+         * @param screen    The screen this keystroke was performed in.
+         * @param keyCode   The key code.
+         * @param scanCode  The raw keyboard scan code.
+         * @param modifiers Additional modifiers.
+         * @return Any other result than {@link InteractionResult#PASS} leads to the cancellation of the key press.
+         */
         InteractionResult keyPressed(Minecraft client, Screen screen, int keyCode, int scanCode, int modifiers);
     }
     
     interface KeyReleased {
+        /**
+         * Invoked whenever a held key is released inside a screen.
+         * Equivalent to Forge's {@code GuiScreenEvent.KeyboardKeyReleasedEvent} event.
+         *
+         * <p> This event is handled in two phases PRE and POST, which are invoked
+         * before and after the keys have been processed by the screen, respectively.
+         *
+         * @param client    The Minecraft instance performing it.
+         * @param screen    The screen this keystroke was performed in.
+         * @param keyCode   The key code.
+         * @param scanCode  The raw keyboard scan code.
+         * @param modifiers Additional modifiers.
+         * @return Any other result than {@link InteractionResult#PASS} leads to the cancellation of the key release.
+         */
         InteractionResult keyReleased(Minecraft client, Screen screen, int keyCode, int scanCode, int modifiers);
     }
     
     interface KeyTyped {
+        /**
+         * Invoked whenever a character is typed within a screen.
+         * Equivalent to Forge's {@code GuiScreenEvent.KeyboardCharTypedEvent} event.
+         *
+         * <p> This event is handled in two phases PRE and POST, which are invoked
+         * before and after the keys have been processed by the screen, respectively.
+         *
+         * @param client    The Minecraft instance performing it.
+         * @param screen    The screen this keystroke was performed in.
+         * @param character The typed character.
+         * @param keyCode   The key code.
+         * @return Any other result than {@link InteractionResult#PASS} leads to the cancellation of the key release.
+         */
         InteractionResult charTyped(Minecraft client, Screen screen, char character, int keyCode);
     }
     
     interface MouseScrolled {
+        /**
+         * Invoked whenever the mouse scroll wheel is moved.
+         * Equivalent to Forge's {@code GuiScreenEvent.MouseScrollEvent} event.
+         *
+         * <p> This event is handled in two phases PRE and POST, which are invoked
+         * before and after the keys have been processed by the screen, respectively.
+         *
+         * @param client The Minecraft instance performing it.
+         * @param screen The screen this keystroke was performed in.
+         * @param mouseX The scaled x-coordinate of the mouse cursor.
+         * @param mouseY The y-coordinate of the mouse cursor.
```

Comment:
The scaled y

Please apply this to every mouse x and mouse y coordinate if applicable, since there's a version with non scaled coords.

Reply:
Done, let me know if I missed any!